### DISCUSSION TITLE
Alert nuovo voto nella homepage

### DISCUSSION THREAD

**Discussion by marcxzz** [iisgiua/giuaschool#796] [Idee e proposte]
Visualizzare un alert (come "è presenta un compito per domani") per la pubblicazione di un nuovo voto.

**Comment by trinko** [comment]
Penso sia meglio l'invio di una notifica. Vorrei espandere le azioni che inviano notifiche, quando avrò il tempo di farlo...

**Reply by marcxzz** [reply]
In aggiunta, ho notato con alcune persone che il bot telegram non funziona
